integration_tests/src/main/python/cast_test.py

```diff
@@ -97,7 +97,7 @@ def test_cast_string_date_invalid_ansi_before_320():
               'spark.sql.ansi.enabled': 'true'}, )
 
 # test databricks, ANSI mode, all databricks versions supports Ansi mode when casting string to date
-@pytest.mark.skipif(not is_databricks91_or_later(), reason="Spark versions(< 320) not support Ansi mode when casting string to date")
+@pytest.mark.skipif(not is_databricks104_or_later(), reason="Spark versions(< 320) not support Ansi mode when casting string to date")
 @pytest.mark.parametrize('invalid', invalid_values_string_to_date)
 def test_cast_string_date_invalid_ansi_databricks(invalid):
```

Comment:
Seems like this test is now redundant, should be covered by test_cast_string_date_invalid_ansi below.

Reply:
removed this test